data/json/items/chemicals_and_resources.json

```diff
@@ -542,6 +558,80 @@
     "container": "bottle_plastic_small",
     "count": 100
   },
+  {
+    "type": "AMMO",
+    "id": "chem_lithium",
+    "category": "chems",
+    "price": 600,
+    "name": "lithium",
+    "name_plural": "lithium",
+    "symbol": "=",
+    "color": "white",
+    "description": "A piece of lithium.  Isolated from a mixture of lithium chloride and potassium chloride, it is a volatile element that is commonly used in the making of heat-resistant glass and ceramics as well as lithium batteries.",
+    "material": "powder_nonflam",
+    "volume": "25 ml",
+    "weight": "134 mg",
+    "//": "density: 0.534 g/cm3",
+    "bashing": 1,
+    "ammo_type": "components",
+    "container": "bottle_plastic_small",
```

Comment:
Plastic bottle is not the safest way of lithium storage. Due to the low density and high flammability of lithium it should be stored in light mineral oil. Though I'm not sure if we have anything like that in game currently.

Reply:
Good point. I'll be adding in mineral oil when I get to adding more stuff to this PR.

Reply:
Best way to do it would be to have lithium item be a liquid that is actually lithium submerged in appropriate amount of some liquid, I'd think.